test/email-field-validation.html

```diff
@@ -0,0 +1,56 @@
+<!doctype html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>vaadin-email-field validation tests</title>
+
+  <script src="../../web-component-tester/browser.js"></script>
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <link rel="import" href="../vaadin-email-field.html">
+  <link rel="import" href="helpers.html">
+</head>
+
+<body>
+<script>
+  describe(`initial validation`, () => {
+    let field, validateSpy;
+
+    beforeEach(() => {
+      field = document.createElement('vaadin-email-field');
+      validateSpy = sinon.spy(field, 'validate');
+    });
+
+    afterEach(() => {
+      field.remove();
+    });
+
+    it('should not validate by default', async() => {
```

Comment:
```suggestion
    it('should not validate by default', async () => {
```

Reply:
The linter complains: error  Unexpected space before function parentheses  space-before-function-paren
I checked the other files and the standard seems to be function(), async()

Reply:
Ah, I didn't notice that. It is fine then 👍